File: Python/Hardware.py
from Product import Product


class Hardware(Product):


    def __init__(self, brand= 0, model= 0):
        self.brand = brand
        self.model = model
        
    def set_brand(self, brand):
        self.brand = brand    

    def set_model(self, model):
        self.model = model

    
    def get_brand(self):
        return self.brand

    def get_model(self):
        return self.model

File: PHP/Hardware.php
<?php

class Hardware extends Product{
    private $brand;
    private $model;
    
   

    public function __construct($brand= 0, $model = 0) {
        $this->brand = $brand;
        $this->model = $model;

    }

    public function setBrand($brand) { //
        $this->brand = $brand;
    }
    public function getBrand() { 
        return $this->brand;
    }

    public function setModel($model){ //
        $this->model = $model;
    }

    public function getModel(){ 
        return $this->model;

    } 

}
?>